tests/cpp/multidevice.cpp

```diff
@@ -33,6 +42,34 @@ MultiDeviceTest::~MultiDeviceTest() {
   }
 }
 
+void MultiDeviceTest::waitForDebugger() {
+  char* rank_to_debug_str = getNvFuserEnv("MPI_DEBUG_RANK");
```

Comment:
what do you think of having a way to make all ranks wait? E.g. by setting the variable to -1

Reply:
I am not sure the word "MPI" should appear explicitely here, since we don't exactly rely on MPI. Also, I point out that the other options in the multidevice test start with the prefix `MULTIDEVICE_`.
Maybe something like `MULTIDEVICE_WAIT_DEBUGGER_AT_RANK`?

Reply:
> Maybe something like MULTIDEVICE_WAIT_DEBUGGER_AT_RANK?

SG! 

> what do you think of having a way to make all ranks wait? E.g. by setting the variable to -1

I don't need that now but I'm happy to see future PRs addressing that! I'd also consider to specify a subset of ranks (e.g. "1,2,4") and to specify all ranks as a special string (e.g. "all").